src/modules/account/types.ts

```diff
@@ -0,0 +1,38 @@
+import { TSMap } from 'typescript-map'
+import { TransferState } from './RadixTransferAccountSystem'
+import { TokenDefinitionState } from './RadixTokenDefinitionAccountSystem'
+
+export type LedgerState = {
+    [key: string]: AccountState
+}
+
+export interface AccountState extends TransferState, TokenDefinitionState {}
+
+export const createInitialState = (): AccountState => {
+    return {
+        spentConsumables: undefined,
+        unspentConsumables: undefined,
+        balance: undefined,
+        tokenUnitsBalance: undefined,
+        tokenDefinitions: undefined
+    }
+}
+
+export enum AtomOperation {
+    STORE,
+    DELETE
+}
+
+export enum TokenType {
+    FIXED,
+    MUTABLE,
+    UNALLOCATED
+}
+
+export function concatMaps(map1: TSMap<any, any>, map2: TSMap<any, any>): TSMap<any, any> {
```

Comment:
Not sure types.ts is the right place for this. There is a RadixUtils class for globally useful functions